.circleci/config.yml

```diff
@@ -97,14 +97,14 @@ workflows:
             - unit
           filters:
             branches:
-              only: master
+              only: main
```

Comment:
We overlooked this in our other repos but this should be fcs, or deleted since only fcs deploys to sandbox now.